Formula/brew-php-switcher.rb

```diff
@@ -0,0 +1,27 @@
+class BrewPhpSwitcher < Formula
+  desc "Switch Apache / Valet / CLI configs between PHP versions"
+  homepage "https://github.com/philcook/php-switcher"
+  url "https://github.com/philcook/brew-php-switcher/archive/v2.0.tar.gz"
+  sha256 "c2303b1b1a66ee90ed900c3beabacd6aa4e921dbcad5242e399c45d86899bc88"
+  head "https://github.com/philcook/brew-php-switcher.git"
+
+  def install
+    bin.install "phpswitch.sh"
+    sh = libexec + "brew-php-switcher"
+    sh.write("#!/usr/bin/env sh\n\nsh #{bin}/phpswitch.sh $*")
```

Comment:
Why is this script needed?

Reply:
I just copied this from the original recipe that was in homebrew/php To maintain backwards compatibility for those that used the script in the past. I didn’t name the script phpswitch.sh. I’m assuming this is here as a wrapper to match the name `brew-php-switcher`.

Reply:
Perhaps it’s better to leave it now, and update the brew-php-switcher to match the name and then update the formula again once it’s renamed? @philcook what do you think? This is your script.